src/site/twitch.tv/modules/emote-menu/EmoteMenuModule.vue

```diff
@@ -115,6 +122,18 @@ export const config = [
 		label: "",
 		defaultValue: new Set(),
 	}),
+	declareConfig<EmoteMenuSortPropertyKey>("ui.emote_menu.sort_emotes_by", "DROPDOWN", {
+		path: ["Appearance", "Interface"],
+		label: "Emote Menu: Sort emotes by property",
```

Comment:
nit: shorten the label

```suggestion
		label: "Emote Menu: Sorting",
		hint: "Define how emotes are sorted in the Emote Menu",
```